share/telemetry.py

```diff
@@ -373,54 +388,59 @@ def function_started_telemetry(ctx: FunctionContext) -> None:
     telemetry_queue.put(FunctionStartedEvent(ctx))
 
 
-def function_ended_telemetry(
-    exception_ignored: bool = False, exception_raised: bool = False, to_be_continued: bool = False
-) -> None:
-    """Triggers the `FunctionEndedEvent` telemetry event."""
-    if not is_telemetry_enabled():
-        return
-
-    with_exception = None
-    if exception_ignored:
-        with_exception = WithExceptionTelemetryEnum.EXCEPTION_IGNORED
-    elif exception_raised:
-        with_exception = WithExceptionTelemetryEnum.EXCEPTION_RAISED
-
-    telemetry_command = FunctionEndedEvent(with_exception=with_exception, to_be_continued=to_be_continued)
-    telemetry_queue.put(telemetry_command)
-
-
-def input_has_output_type_telemetry(input_id: str, output_type: str) -> None:
+# def function_ended_telemetry(
```

Comment:
minor: not sure if we need to introduce all this comments in order to not have the code running

for the final outcome of sending data to the telemetry only once at, just after parsing the config file, we can keep the code and change just https://github.com/elastic/elastic-serverless-forwarder/pull/156/commits/ee0b5a88bf8a63b4fd706121b3156d4c5059fcb4#diff-6fde71d82d3ead64aa933407ae918d5d91fbb4be69e09f236c2d5579c2cdb733R349

Reply:
I commented out a bunch of code to highlight the impact area. It can be handy to keep it around until we complete the review iteration.

I am okay with keeping the code around (uncommented) since this is primarily a telemetry phase 1, and we may go back to it later. However, usually, I remove unused code, even when it is inconvenient going back to previous PRs.

Reply:
> However, usually, I remove unused code

that's my preferred option as well :)